packages/ubuntu_wizard/lib/src/widgets/wizard_page.dart

```diff
@@ -88,9 +92,12 @@ class _WizardPageState extends State<WizardPage> {
     final currentStep = (wizardScope?.routeData as int?);
 
     // a leading (back) action only on pages that have more than one action
-    final leading = widget.actions.length > 1 ? widget.actions.first : null;
+    final leading = widget.actions.length > 1 || widget.singleLeading
```

Comment:
let's refactor the whole bottom bar out of WizardPage when it's appropriate to start refactoring again :)